### PR TITLE
[Docs] Add Change log and other edits

### PR DESCRIPTION
**Pull request by jillnogold**

### CODE REVIEW

File: docs/change-log/_index.md [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+- When selecting **All** in the **Tag** dropdown list of the **Projects | Feature Store | Feature Vectors** tab, 
+all of the vectors display correctly. [View in Git](https://github.com/mlrun/ui/pull/1549).
+- MLRun API returns the correct `next_run_time` value. [View in Git](https://github.com/mlrun/mlrun/pull/2862).
+- Filtering in **Projects | Jobs | Monitor Workflows** returns correct results for all ranges. (Previously was not accurate 
+for > 1 hour.) [View in Git](https://github.com/mlrun/mlrun/pull/2786).
+- The updated tooltips in **Projects | Jobs | Schedule** explain that day 0 (for cron jobs) is Monday, and not Sunday. 
+[View in Git](https://github.com/mlrun/ui/pull/1571).
+- When creating a project, the error `NotImplementedError` was updated to explain that MLRun does not have a 
+DB to connect to. [View in Git](https://github.com/mlrun/mlrun/pull/2856).
+- Previewing artifacts in the UI was fixed. [View in Git](https://github.com/mlrun/mlrun/pull/2592).
```
Comment (Tankilevitch) [inline docs/change-log/_index.md:35]: Its hard to understand what is being fixed, e.g I would change 
MLRun API returns the correct `next_run_time` value -> Fix not updating `next_run_time` when skipping scheduling due to concurrent runs

In my opinion this way we explain what was fixed and not what is working.

File: docs/change-log/_index.md [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+- Supports ingesting Avro-encoded Kafka records. [View in Git](https://github.com/mlrun/mlrun/issues/2649).
+
+### Closed issues
+- When running a workflow with `local=True`, the serving function is updated. [View in Git](https://github.com/mlrun/mlrun/issues/2716).
+- The Kubernetes **Pods** tab in **Monitor Workflows** now shows the complete pod details. [View in Git](https://github.com/mlrun/mlrun/pull/1576).
+- When selecting **All** in the **Tag** dropdown list of the **Projects | Feature Store | Feature Vectors** tab, 
+all of the vectors display correctly. [View in Git](https://github.com/mlrun/ui/pull/1549).
+- MLRun API returns the correct `next_run_time` value. [View in Git](https://github.com/mlrun/mlrun/pull/2862).
+- Filtering in **Projects | Jobs | Monitor Workflows** returns correct results for all ranges. (Previously was not accurate 
+for > 1 hour.) [View in Git](https://github.com/mlrun/mlrun/pull/2786).
```
Comment (Tankilevitch) [inline docs/change-log/_index.md:30]:
```suggestion
- Fix not accurate view when filtering for > 1 hour in **Projects | Jobs | Monitor Workflows** [View in Git](https://github.com/mlrun/mlrun/pull/2786).
```

File: docs/change-log/_index.md [hunk 3]
```diff
@@ -0,0 +1,339 @@
...
+### New and updated features
+
+#### Feature store
+- Supports ingesting Avro-encoded Kafka records. [View in Git](https://github.com/mlrun/mlrun/issues/2649).
+
+### Closed issues
+- When running a workflow with `local=True`, the serving function is updated. [View in Git](https://github.com/mlrun/mlrun/issues/2716).
+- The Kubernetes **Pods** tab in **Monitor Workflows** now shows the complete pod details. [View in Git](https://github.com/mlrun/mlrun/pull/1576).
+- When selecting **All** in the **Tag** dropdown list of the **Projects | Feature Store | Feature Vectors** tab, 
+all of the vectors display correctly. [View in Git](https://github.com/mlrun/ui/pull/1549).
```
Comment (Tankilevitch) [inline docs/change-log/_index.md:27]:
```suggestion
- Fix UI crash when selecting **All** in the **Tag** dropdown list of the **Projects | Feature Store | Feature Vectors** tab. [View in Git](https://github.com/mlrun/ui/pull/1549).
```

File: docs/change-log/_index.md (hunk 1 above)
Comment (Tankilevitch) [inline docs/change-log/_index.md:35]: IMO this is too broad, we need to focus on which case exactly, IIUC in this fixes specific case of previewing dir artifacts, so I would mention that.

File: docs/change-log/_index.md [hunk 4]
```diff
@@ -0,0 +1,339 @@
...
+- MLRun API returns the correct `next_run_time` value. [View in Git](https://github.com/mlrun/mlrun/pull/2862).
+- Filtering in **Projects | Jobs | Monitor Workflows** returns correct results for all ranges. (Previously was not accurate 
+for > 1 hour.) [View in Git](https://github.com/mlrun/mlrun/pull/2786).
+- The updated tooltips in **Projects | Jobs | Schedule** explain that day 0 (for cron jobs) is Monday, and not Sunday. 
+[View in Git](https://github.com/mlrun/ui/pull/1571).
```
Comment (Tankilevitch) [inline docs/change-log/_index.md:32]:
```suggestion
- Update tooltips in **Projects | Jobs | Schedule** to explain that day 0 (for cron jobs) is Monday, and not Sunday) . 
[View in Git](https://github.com/mlrun/ui/pull/1571).
```

File: docs/change-log/_index.md [hunk 5]
```diff
@@ -0,0 +1,339 @@
...
+
+## v1.2.1
+
+### New and updated features
+
+#### Feature store
+- Supports ingesting Avro-encoded Kafka records. [View in Git](https://github.com/mlrun/mlrun/issues/2649).
+
+### Closed issues
+- When running a workflow with `local=True`, the serving function is updated. [View in Git](https://github.com/mlrun/mlrun/issues/2716).
```
Comment (Tankilevitch) [inline docs/change-log/_index.md:24]: I don't understand what it fixes

File: docs/change-log/_index.md [hunk 6]
```diff
@@ -0,0 +1,339 @@
...
+- The Kubernetes **Pods** tab in **Monitor Workflows** now shows the complete pod details. [View in Git](https://github.com/mlrun/mlrun/pull/1576).
```
Comment (Tankilevitch) [inline docs/change-log/_index.md:25]:
```suggestion
- Fix Kubernetes **Pods** tab in **Monitor Workflows** didn't show the complete pod details. [View in Git](https://github.com/mlrun/ui/pull/1576).
```

**Review by Tankilevitch — CHANGES_REQUESTED** [review]

File: docs/change-log/_index.md (hunk 5 above)
Comment (jillnogold) [reply docs/change-log/_index.md:24]: Pls see https://jira.iguazeng.com/browse/ML-3141 - and add a description. Tnx

File: docs/data-prep/ingesting_data.md [hunk 7]
```diff
@@ -87,9 +87,9 @@ additional details and code-samples:
 
 <!---
 TODO - add this once we have Spark service documentation.
-1. [Spark service](???) - **do we have a page for this? Are we documenting it?**
+- [Spark service](???) - **do we have a page for this? Are we documenting it?**
```
Comment (gilad-shaham) [inline docs/data-prep/ingesting_data.md:90]: No need for the comment, we are focusing on spark operator

**Review by gilad-shaham — APPROVED** [review]

**Pull request merged**